src/com/sun/jna/CallbackReference.java

```diff
@@ -67,9 +67,9 @@
             throw new Error("Error looking up CallbackProxy.callback() method");
         }
     }
-    
+
     private static final Class<?> DLL_CALLBACK_CLASS;
-    
+
```

Comment:
Sorry about this - I did not wait for travis and thus this got in via #1280

Reply:
Another PR is trying to fix it too.   And not having to wait for travis is the whole purpose of this PR! 😃